sklearn/ensemble/_gb.py

```diff
@@ -972,19 +984,23 @@ class GradientBoostingClassifier(ClassifierMixin, BaseGradientBoosting):
     GB builds an additive model in a
     forward stage-wise fashion; it allows for the optimization of
     arbitrary differentiable loss functions. In each stage ``n_classes_``
-    regression trees are fit on the negative gradient of the
-    binomial or multinomial deviance loss function. Binary classification
+    regression trees are fit on the negative gradient of the loss function,
+    e.g. binary or multiclass log loss. Binary classification
     is a special case where only a single regression tree is induced.
 
     Read more in the :ref:`User Guide <gradient_boosting>`.
 
     Parameters
     ----------
-    loss : {'deviance', 'exponential'}, default='deviance'
-        The loss function to be optimized. 'deviance' refers to
-        deviance (= logistic regression) for classification
-        with probabilistic outputs. For loss 'exponential' gradient
-        boosting recovers the AdaBoost algorithm.
+    loss : {'log_loss', 'exponential'}, default='log_loss'
+        The loss function to be optimized. 'log_loss' refers to binomial and
+        multinomial deviance, the same as used in linear logistic regression.
```

Comment:
To be consistent with the docstring change above this one, can we remove the reference to "deviance"?

```suggestion
        multinomial log loss, the same as used in logistic regression.
```

(Also I think most think of logistic regression as linear model.)

Reply:
Hmm. How about calling them just binomial and multinomial log-likelihood?

Reply:
"binomial" and "multinomial" are distributions, have a likelihood and, equivalently, a deviance. Log loss, although derivable from those likelihoods, is a loss that a priori has no distribution. I would speak of binary and multiclass log loss.